app/assets/stylesheets/content/_tooltips.md

```diff
@@ -107,6 +107,29 @@ These can contain simple texts but are not suitable for HTML within the Tooltip.
 
 Note that the tabindex has to be set manually on the `<span>` and not the containing element. `tabindex="0"` makes the item tabbable at all.
 
+### HTML tooltips
+
+```
+<div class="form--field">
+  <label class="form--label" for="new_password" title="New password">New password</label>
+  <div class="form--field-container tooltip-visible">
+    <span class="form--text-field-container">
+      <input class="form--text-field -password" id="new_password" name="new_password" size="25" type="password">
```

Comment:
What if the id is not `new_password`? (or, to be precise, doesn't begin with `new_password`?)